package.json

```diff
@@ -12,9 +12,11 @@
     "dotenv": "^10.0.0",
     "express": "~4.16.1",
     "http-errors": "~1.6.3",
+    "mongodb": "^3.6.9",
+    "mongoose": "^5.13.0",
     "morgan": "~1.9.1",
     "node-fetch": "^2.6.1",
-    "pug": "2.0.0-beta11"
+    "pug": "^3.0.2"
```

Comment:
this is service for api only, do we need `pub` anymore? should we cleanup all the templates as well?